apps/demo-react/hooks/useWithdrawalsData.ts

```diff
@@ -198,7 +198,7 @@ export const useWithdrawalRequestMethods = () => {
   );
 
   const steth = useCallback(
-    async ({ requests }) => {
+    async ({ requests }: { requests: any }) => {
```

Comment:
`unknown` if you really don't know about type

Reply:
fixed